renovate.json5

```diff
@@ -69,6 +69,34 @@
         {
             "matchUpdateTypes": ["major"],
             "labels": ["major", "dependencies"]
+        },
+
+        {
+            // Can't upgrade because: > v5 doesn't support Prompts, so we don't want to upgrade atm
```

Comment:
I think react-router offers an API for this. useBlocker` API can block navigation on certain conditions.  I don't know our comet's implementation details and usage of react router `Prompt` in depth, but I am pretty sure, it should be possible since react-router [Version 6.7.0 with unstable_useblocker](https://reactrouter.com/start/changelog#v670). 

`UseBlocker` Feature should already be stable in latest react router version.

Reply:
Ah true, then we can try that. I must have missed it